corehq/apps/commtrack/static/commtrack/ko/locations.js

```diff
@@ -44,7 +25,7 @@ function LocationSettingsViewModel() {
 
     this.new_loctype = function() {
         var parent_pk = (_.last(settings.loc_types()) || {}).pk;
-        var new_loctype = new LocationTypeModel({parent_type: parent_pk}, this);
+        var new_loctype = new LocationTypeModel({parent_type: parent_pk}, commtrack_enabled);
```

Comment:
nice this is looking much better to me - thanks for being patient!

(we should merge the rest of this even if the functionality ends up not being required to change)